### PR TITLE
[docs content] MDX layouts

### PR DESCRIPTION
**Pull request by sarah11918**
## Changes

Adds a short section explicitly on layouts to the `@astrojs/mdx` README to have a place for Support Squad to point to. Feel free to edit to taste, or to PR later, but we're starting to get this question specifically about layouts A LOT now. So having *something* we can just link to is helpful, and I'm pretty sure this is at least working and correct.  😅 

- includes `export {default} from '../../layouts/BaseLayout.astro';` option, with link to [MDX docs on layouts](https://mdxjs.com/docs/using-mdx/#layout) for layouts that do not need to receive props.

- includes and gives an example of importing and wrapping MDX content in a `<Layout {...frontmatter}>` component, passing all frontmatter props to the layout.

## Testing

Not tested. For Docs.
## Docs

Will automatically update integration docs.

### CODE REVIEW

File: packages/integrations/mdx/README.md [hunk 1]
```diff
@@ -134,6 +134,47 @@ const posts = await Astro.glob('./*.mdx');
 ))}
 ```
 
+### Layouts
+
+You can use the [MDX layout component](https://mdxjs.com/docs/using-mdx/#layout) to specify a layout component to wrap all page content. This is done with a default export statement at the end of your `.mdx` file:
```
Comment (aFuzzyBear) [inline packages/integrations/mdx/README.md:139]:
```suggestion
You can `import` your Astro Layouts to your `.mdx` files, to wrap around all of the page content. This is done by placing an import statement at the top of the file. Underneath any frontmatter that maybe at the top of the page
```

File: packages/integrations/mdx/README.md [hunk 2]
```diff
@@ -134,6 +134,47 @@ const posts = await Astro.glob('./*.mdx');
...
+
+```mdx
+// src/pages/my-page.mdx
+
+export {default} from '../../layouts/BaseLayout.astro';
```
Comment (aFuzzyBear) [inline packages/integrations/mdx/README.md:144]:
```suggestion
import BaseLayout from '../../layouts/BaseLayout.astro';
```

**Review by aFuzzyBear — COMMENTED** [review]
I found the export statement when I had followed that advice originally to be problematic, after testing out how layouts are brought into mdx, I would suggest that we apply the following changes to make the behavior on par with expectations.

**Comment by sarah11918** [comment]
@aFuzzyBear The export statement worked fine for me (when no props to pass), and there's a second example of the wrapping `<Layout />` component method already included below.

**Comment by aFuzzyBear** [comment]
how very peculiar, when I first tired it I didn't get much luck,

**Review by matthewp — APPROVED** [review]
lgtm

**Pull request merged**